### PR TITLE
added remove.me

### PR DESCRIPTION
**Pull request by jdunk**
Opening this test PR

### CODE REVIEW

File: remove.me [hunk 1]
```diff
@@ -1,4 +1,3 @@
 adding some lines here
-so that they
-will show up in
-the pull request
+so that they will show up
```
Comment (jdunk) [inline remove.me:2]: is this line too long now?

File: remove.me [hunk 2]
```diff
@@ -1,4 +1,3 @@
 adding some lines here
-so that they
-will show up in
-the pull request
```
Comment (jdunk) [inline remove.me]: this is a comment on a removed line

File: remove.me [hunk 3]
```diff
@@ -0,0 +1,3 @@
+adding some lines here
+so that they will show up
```
Comment (jdunk) [inline remove.me:2]: this is a comment made on the PR, not on the individual commit

File: .provision/provision.sh [hunk 4]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 # fix ppa and install xdebug
+# and I would've gotten away with it too! >:D >:D >:D
```
Comment (jdunk) [inline .provision/provision.sh]: so evil!

File: remove.me (hunk 3 above)
Comment (jmocko92) [reply remove.me:2]: Okay, looks good man.

Comment (jmocko92) [reply remove.me:2]: Thought I have one idea tk.

Comment (jmocko92) [reply remove.me:2]: Here's another comment

**Review by jmocko92 — APPROVED** [review]

**Comment by jmocko92** [comment]
I just came up with an idea for a different change though

File: .provision/provision.sh (hunk 4 above)
Comment (jmocko92) [reply .provision/provision.sh]: Yes I will start a review here. Getting the hang of this now

Comment (jmocko92) [reply .provision/provision.sh]: And here's another comment

**Review by jmocko92 — CHANGES_REQUESTED** [review]

**Comment by jdunk** [comment]
ok, ready for your re-review

**Review by jmocko92 — APPROVED** [review]

**Pull request merged**